hack/reduce-snapshot.sh

```diff
@@ -65,4 +65,4 @@ if [ "${SINGLE_COMPONENT}" == "true" ]; then
 fi
 
 # we need to create snapshot file to be passed to later stages.
-echo "${SNAPSHOT}" | jq '.' | tee "${SNAPSHOT_PATH}"
+printf "%s" "${SNAPSHOT}" | jq '.' | tee "${SNAPSHOT_PATH}"
```

Comment:
Interesting. Have you been able to reproduce the issue and verify this indeed fixed it?

Reply:
I can't reproduce it, but from reading up on it, it sounds like `printf` might be helpful here. It's a low risk change, so I thought it would be worth the try. The resource I used. https://www.in-ulm.de/~mascheck/various/argmax/

Reply:
I do wonder if there's something else going on too, because 174 images in a file just doesn't seem like enough to cause a problem. But either way, `printf` _should_ be more efficient.

Reply:
Nice resource! I do think the `printf` option there works only because it's using a null character and pipeline through xargs with a null character limit. It probably won't apply here.

Maybe instead of holding on to the snapshot in a variable it can just be a file. That way we don't have to pipe it to jq. We can just have jq read the file directly.

Anyways, it's worth trying out different things. It would be great to verify that any of these options (including printf) do fix the problem before having to go through the whole release process.

Reply:
After looking at the script better, this all might be unnecessary. We're reading it in from a file and then dumping it back to a file without any changes.

Reply:
Updated to read from a file. PTAL when you can. Thanks!